### ISSUE TITLE
Customized Keyboard Shortcut

### ISSUE THREAD

**Issue by bmart1n** [Liresol/anki-custom-shortcuts#153]
Hey, I have been using 8bit zero2 controller for a while on my windows laptop for Anki, but now, after a few months of not using it, it doesn't seem to pair accordingly to my customized shortcuts. I tested out to check the letters (changing 1/2/3/4 to j/g/h/i) but they do not seem to pair to shortcuts they were assigned to. Any tips for fixing this issue?

**Comment by Liresol** [comment]
Hello,
Just to make sure, you've restarted Anki so that the changes take effect and tested whether just using the keyboard that the shortcuts work (which they might not based on what you've said). I also see that `i` is used there, which corresponds to an existing reviewer default ("reviewer card info"), but you probably already changed that shortcut to something else.
If you've already tried stuff like this, could you post which shortcuts you've changed in case there something unusual about your config?

**Comment by bmart1n** [comment]
Yes, once I made the changes to the keyboard shortcuts I closed and re-opened Anki. I also did a double-check to make sure that any of the configurations did not overlap with other shortcut commands. I also tried using the shortcuts with my laptop's keyboard first and then with the controller, but I had no luck with either. 

These are the shortcuts I changed to try to match to my controller:

"m_toolbox create filtered deck": "F", -> "L"
"m_toolbox undo": "Ctrl+Z", ----------> "K"
"main deckbrowser": "D", -------------> "N"

"reviewer card info": "I", ---------------> "U"
"reviewer choice 1": "1", ---------------> "J"
"reviewer choice 2": "2", ---------------> "G"
"reviewer choice 3": "3", ---------------> "H"
"reviewer choice 4": "4", ---------------> "I"
"reviewer edit current": "E", -----------> "Z"
"reviewer flip card 1": " ", -------------> "M"
"reviewer more options": "M", --------> "P"
"reviewer options menu": "o", --------> "w"

 "Ω custom paste end style": "n", -----> "q"

**Comment by Liresol** [comment]
Sorry for the delay.
I have to say that this issue is looking quite weird, since these changes act fine on my computer.
The only things I can think of right now are that the issue could come from having an old version of Anki (unlikely to change anything but it's possible there are other issues) or having conflicts from a previous shortcut setting that went unnoticed (also unlikely but I know there are some nontrivial bad interactions) or an addon conflict that I'm not aware of.
I think I might need to see your Anki version and possibly other addons to figure this out.

If you have the time/energy, it might also be informative to see whether your settings react to the defaults instead of the customized settings (e.g. seeing if your current settings still use "1" for reviewer choice 1 or whether they seem to not react at all) since that info could also help figure out what's going on behind the scenes.

As an aside, `"Ω custom paste end style": "q"` doesn't do anything on its own, since it's not actually a shortcut. If you use the custom paste functionality (which doesn't seem to be the case), then addon this to "y" would tell the addon contain the custom formatting in any pasted texts inside the paste (e.g. a pasted bolded text would not cause the following typed characters to be bold).
